SQLite/Core/Connection.swift

```diff
@@ -67,10 +268,32 @@ public final class Connection {
     ///     Default: `false`.
     ///
     /// - Returns: A new database connection.
-    public init(_ location: Location = .InMemory, readonly: Bool = false) throws {
+    public convenience init(_ location: Location = .InMemory, readonly: Bool = false, vfsName: String? = nil) throws {
```

Comment:
Looks like the function docs above need to be updated to include the `vfsName` parameter.